lua/setup.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+check_exists_and_create_symlink() {
```

Comment:
Do not indent using Tabs, indent using spaces.

Place curly bracket on a newline.